td-logger.gemspec

```diff
@@ -32,8 +32,10 @@ EOF
   gem.require_paths = ['lib']
 
   gem.add_dependency "msgpack", ">= 0.5.6", "< 2.0"
-  gem.add_dependency "td-client", ">= 0.8.66", "< 2.0"
+  gem.add_dependency "td-client", ">= 1.0.8", "< 3.0"
   gem.add_dependency "fluent-logger", ">= 0.5.0", "< 2.0"
+  gem.add_dependency "mutex_m", ">= 0.2.0", "< 1.0"
+  gem.add_dependency "logger"
```

Comment:
Added logger gem to follow up Ruby 3.5+ behavior.
```
/home/runner/work/td-logger-ruby/td-logger-ruby/spec/td/logger/agent/config_spec.rb:3: warning: logger used to be loaded from the standard library, but is not part of the default gems since Ruby 3.5.0.
```